### ISSUE TITLE
[onert] Armv7l cross build generates too many warnings

### ISSUE THREAD

**Issue by periannath** [Samsung/ONE#5173]
Recent armv7l cross build produces too many warnings related to externals.

#### clog warning

```bash
/home/jhyeo/ONE_alpha/externals/CPUINFO/deps/clog/src/clog.c: In function ‘clog_vlog_fatal’:                                                                                                                  
/home/jhyeo/ONE_alpha/externals/CPUINFO/deps/clog/src/clog.c:112:4: warning: ignoring return value of ‘write’, declared with attribute warn_unused_result [-Wunused-result]                                   
    write(STDERR_FILENO, out_buffer, prefix_chars + format_chars + CLOG_SUFFIX_LENGTH);                                                                                                                       
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

#### pthreadpool warning
```bash
In file included from /home/jhyeo/ONE_alpha/externals/PTHREADPOOL/src/pthreads.c:50:                                                                                                                                
/home/jhyeo/ONE_alpha/externals/PTHREADPOOL/include/pthreadpool.h:997:2: warning: ‘pthreadpool_function_1d_t’ is deprecated [-Wdeprecated-declarations]                                                             
  pthreadpool_function_1d_t function,                                                                                                                                                                               
  ^~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                         
/home/jhyeo/ONE_alpha/externals/PTHREADPOOL/include/pthreadpool.h:1003:2: warning: ‘pthreadpool_function_1d_tiled_t’ is deprecated [-Wdeprecated-declarations]                                                      
  pthreadpool_function_1d_tiled_t function,                                                                                                                                                                         
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                   
/home/jhyeo/ONE_alpha/externals/PTHREADPOOL/include/pthreadpool.h:1010:2: warning: ‘pthreadpool_function_2d_t’ is deprecated [-Wdeprecated-declarations]                                                            
  pthreadpool_function_2d_t function,                                                                                                                                                                               
  ^~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                         
/home/jhyeo/ONE_alpha/externals/PTHREADPOOL/include/pthreadpool.h:1017:2: warning: ‘pthreadpool_function_2d_tiled_t’ is deprecated [-Wdeprecated-declarations]                                                      
  pthreadpool_function_2d_tiled_t function,                                                                                                                                                                         
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                   
/home/jhyeo/ONE_alpha/externals/PTHREADPOOL/include/pthreadpool.h:1026:2: warning: ‘pthreadpool_function_3d_tiled_t’ is deprecated [-Wdeprecated-declarations]                                                      
  pthreadpool_function_3d_tiled_t function,                                                                                                                                                                         
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                   
/home/jhyeo/ONE_alpha/externals/PTHREADPOOL/include/pthreadpool.h:1037:2: warning: ‘pthreadpool_function_4d_tiled_t’ is deprecated [-Wdeprecated-declarations]                                                      
  pthreadpool_function_4d_tiled_t function,                                                                                                                                                                         
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~   
```

#### xnnpack warning
```bash
[ 97%] Building ASM object externals/XNNPACK/CMakeFiles/XNNPACK.dir/src/f32-gemm/4x8-minmax-aarch32-neon-cortex-a53.S.o                                                                                             
cc1: warning: switch -mcpu=cortex-a7 conflicts with -march=armv8-a switch      
```

- pthreadpool warning is repeated more than a few hundread times
- I thinks it would be better to suppress those warnings related to externals

**Comment by periannath** [comment]
#5174 will remove first two warnings. Third one is related to general cmake build flag and I'm not sure how to resolve it.

**Comment by periannath** [comment]
### How to remove third warning

### Tl;dr

- Replace `-mcpu=cortex-a7` in `config_armv7l-linux.cmake` with `-mtune=cortex-a7 -march=armv7ve`

### Explain
- `-mcpu=X` : This flag is a combination of `-march` and `-mtune`
  - Reference : https://community.arm.com/developer/tools-software/tools/b/tools-software-ides-blog/posts/compiler-flags-across-architectures-march-mtune-and-mcpu
- `-mcpu=cortex-a7` implies `-march=armv7ve -mtune=cortex-a7`
- Combination of `march` and `mtune` doesn't produce gcc warning

**Comment by periannath** [comment]
> Replace -mcpu=cortex-a7 in config_armv7l-linux.cmake with -mtune=cortex-a7 -march=armv7ve

I'll not apply this solution since its side effect is unclear. Some documents say they are the same, but other documents say they might produce a different binary.
